springwolf-core/src/main/java/io/github/stavshamir/springwolf/asyncapi/scanners/channels/AbstractChannelScanner.java

```diff
@@ -86,14 +86,16 @@ private Map.Entry<String, ChannelItem> mapMethodToChannel(Method method) {
         Map<String, ? extends ChannelBinding> channelBinding = buildChannelBinding(annotation);
         Map<String, ? extends OperationBinding> operationBinding = buildOperationBinding(annotation);
         Class<?> payload = getPayloadType(method);
-        ChannelItem channel = buildChannel(channelBinding, payload, operationBinding);
+        String operationId = channelName + "-" + method.getName();
```

Comment:
The AbstractChannelScanner cannot infer whether it is used used by a publisher or subscriber.
Currently it is only used by RabbitChannelsScanner and MethodLevelKafkaListenerScanner (both consumers) in public APIs.

Does the interface `getListenerClassName` indicate it will only be used by consumers? If so, I suggest adding `subscribe` to the operationId and renaming the class to AbstractListenerScanner. And then also renaming RabbitChannelsScanner to RabbitListenerScanner. Or rather use Consumer in the naming?

In general, I saw also overlaps with the unused AbstractMethodLevelListenerScanner.

Reply:
We should absolutely rename `AbstractChannelScanner` to `AbstractListenerScanner`, I chose that name before we supported producers :)
I prefer `Listener` to `Consumer` because this is also the spring terminology.

Reply:
I updated the PR